homeassistant/components/guardian/switch.py

```diff
@@ -49,13 +34,17 @@ async def async_setup_entry(
     platform = entity_platform.current_platform.get()
 
     for service_name, schema, method in [
-        (SERVICE_DISABLE_AP, None, "async_disable_ap"),
-        (SERVICE_ENABLE_AP, None, "async_enable_ap"),
-        (SERVICE_REBOOT, None, "async_reboot"),
-        (SERVICE_RESET_VALVE_DIAGNOSTICS, None, "async_reset_valve_diagnostics"),
+        (SERVICE_DISABLE_AP, {}, "async_disable_ap"),
+        (SERVICE_ENABLE_AP, {}, "async_enable_ap"),
+        (SERVICE_REBOOT, {}, "async_reboot"),
+        (SERVICE_RESET_VALVE_DIAGNOSTICS, {}, "async_reset_valve_diagnostics"),
         (
             SERVICE_UPGRADE_FIRMWARE,
-            SERVICE_UPGRADE_FIRMWARE_SCHEMA,
+            {
+                vol.Optional(CONF_URL): cv.url,
```

Comment:
No defaults anymore?

Reply:
Correct – see changelog for `aioguardian` bump: https://github.com/bachya/aioguardian/releases/tag/1.0.1